tools/sggolicenses/tools.go

```diff
@@ -112,7 +112,7 @@ func Check(ctx context.Context, disallowedTypes ...string) error {
 }
 
 func PrepareCommand(ctx context.Context) error {
-	_, err := sgtool.GoInstall(ctx, "github.com/google/go-licenses", version)
+	_, err := sgtool.GoInstall(ctx, "github.com/google/go-licenses/v2", version)
```

Comment:
@alethenorio CI fails here due to the `/v2` suffix. I actually opened this old PR to handle this case a while back but we ended up not merging it in (as we never merged mockery into sage either): https://github.com/einride/sage/pull/654

Do you think we should add something like this, or would you prefer we inline the whole installation phase, as we do e.g. here: https://github.com/einride/sage/blob/master/tools/sggolangcilintv2/tools.go#L181-L211